policies-procedures-and-docs/cohort-notes/week-10.md

```diff
@@ -0,0 +1,261 @@
+
```

Comment:
We could probably remove that one as it is already in the cohort repo

Reply:
I looked around a bit to confirm this is present somewhere else before deleting. I haven't found it yet. 👀